docs/pages/machine-workload-identity/mwi_tbot_aws_guide.mdx

```diff
@@ -0,0 +1,326 @@
+  bot_name: example
+  join_method: iam
+  # Restrict the AWS account and (optionally) ARN that can use this token.
+  # This information can be obtained from running the
+  # "aws sts get-caller-identity" command from the CLI.
+  allow:
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+```
+
+Replace:
+
+- `111111111111` with the ID of your AWS account.
+- `teleport-bot-role` with the name of the AWS IAM role you created and assigned
+  to the EC2 instance.
+- `example` with the name of the bot you created in the second step.
+- `i-*` indicates that any instance with the specified role can use the join
+  method. If you wish to restrict this to an individual instance, replace `i-*`
+  with the full instance ID.
+
+
+Use `tctl` to apply this file:
+
+```code
+$ tctl create bot-token.yaml
+```
+
+## Step 4/9. Configure `tbot`
+
+**This step is completed on the AWS EC2 instance.**
+
+Create `/etc/tbot.yaml`:
+
+```yaml
+version: v2
+
+# Teleport proxy server address
+proxy_server: example.teleport.sh:443
+
+# Onboarding configuration
+onboarding:
+  join_method: token
+  token: example-bot
+
+# Storage configuration for tbot state
+storage:
+  type: directory
+  path: /var/lib/teleport/bot
+
+# Outputs section - specifies the credentials tbot should generate
+outputs:
+  - type: identity
+    destination:
+      type: directory
+      path: /opt/machine-id/identity
+
+# Services section - configures the workload-identity-api service
+services:
+  - type: workload-identity-api
+    # Path to the socket file that workloads will connect to
+    listen: unix:///opt/machine-id/workload.sock
+    
+    # Specifies which WorkloadIdentity resource will be used to issue credentials
+    selector:
+      # Replace with the actual name of your Workload Identity resource
+      name: example-workload-identity
+```
+
+Replace:
+
+- `example.teleport.sh:443` with the address of your Teleport Proxy Service or
+  Auth Service. Prefer using the address of a Teleport Proxy Service instance.
+- `example-bot` with the name of the token you created in the second step.
+
+(!docs/pages/includes/machine-id/daemon-or-oneshot.mdx!)
+
+## Step 5/9. Create a role for tbot
+
+To let Teleport users or services run `tbot`, you need a role that grants the required permissions for authenticating and requesting short-lived certificates.
+`tbot` uses a bot token issued by the Auth Service and operates under a bot role that defines what the bot can impersonate (e.g., roles, resources).
+
+Save the following role definition as `tbot-role.yaml`:
+
+```yaml
+kind: role
+version: v7
+metadata:
+  name: tbot
+spec:
+  allow:
+    # Allow tbot to impersonate these roles when issuing certificates.
+    impersonate:
+      roles: ['workload', 'ci-cd']
+    # Allow reading and creating resources required for impersonation and joining.
+    rules:
+      - resources: ['token']
+        verbs: ['create']
```

Comment:
Again, you don't need this. It's actually a little dangerous too I think because it'd allow the bot to create new join tokens (which isn't something you'd ever need to do).